archetype-common/src/test/java/org/apache/maven/archetype/old/ArchetypeTest.java

```diff
@@ -248,14 +250,13 @@ public void testAddModuleToParentPOM()
         StringWriter out = new StringWriter();
         assertTrue( DefaultOldArchetype.addModuleToParentPom( "myArtifactId1", new StringReader( pom ), out ) );
 
-        assertEquals(
-                     "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
-                     + "<project>\n"
-                     + "  <packaging>pom</packaging>\n"
-                     + "  <modules>\n"
-                     + "    <module>myArtifactId1</module>\n"
-                     + "  </modules>\n"
-                     + "</project>", out.toString() );
+        assertEquals( trim( "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"no\"?>\n"
+                + "<project>\n"
```

Comment:
Won't these Unix line endings fail on Windows?

Reply:
No, it does not because the `trim` function turns all elements in one line without new lines.

Reply:
it is our function, not the one from Java.